src/main/java/org/jabref/gui/groups/GroupTree.css

```diff
@@ -76,9 +76,9 @@
     -fx-padding: 0.45em 0.2em 0.45em 0.2em;
 }
 
-.tree-table-row-cell:selected > .tree-table-cell > .tree-disclosure-node > .arrow {
-    -fx-background-color: white;
-}
+/*.tree-table-row-cell:selected > .tree-table-cell > .tree-disclosure-node > .arrow {*/
+    /*-fx-background-color: -jr-white;*/
```

Comment:
In your screenshot, the arrow is never visible accept on hover because it is also white...